packages/connect-plugin-ethereum/tsconfig.libESM.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "./tsconfig.lib.json",
+    "compilerOptions": {
+        "outDir": "libESM",
+        "module": "ESNext"
```

Comment:
I am thinking that it might be good time to also `target` to `esnext` too, because there is no reason we should transpile to older es version

Reply:
Sure, let's do that https://github.com/trezor/trezor-suite/pull/16729/files#diff-97fc10c343bde083841616cc3853cf0d7e98003d6a1b11aa36a7665e69678b0bR6